src/types.ts

```diff
@@ -11,8 +11,8 @@ export interface Block {
 }
 
 export interface DecodedScript {
```

Comment:
Since this will need a refactor everywhere that uses `DecodedScript`, I will create an issue to refactor this and merge this PR as we discussed in Slack

Reply:
https://github.com/HathorNetwork/hathor-wallet-service-sync_daemon/issues/16